coops-ui/tests/unit/Dashboard.spec.ts

```diff
@@ -55,19 +55,19 @@ describe('Dashboard - UI', () => {
 
     expect(wrapper.vm.hasBlockerFiling).toEqual(false)
     expect(wrapper.vm.$el.querySelector('#btn-standalone-addresses')
-      .getAttribute('disabled')).toBeFalsy()
+      .getAttribute('disabled')).not.toBe('true')
     expect(wrapper.vm.$el.querySelector('#btn-standalone-directors')
-      .getAttribute('disabled')).toBeFalsy()
+      .getAttribute('disabled')).not.toBe('true')
```

Comment:
Actually, I think in this case the attribute is undefined. Which is not true. But lots of things are not true. Anyway...

Reply:
What would you like to see here. The code we're changing was originally `toBeFalsy()` & `toBeTruthy()` in master, so i followed the pattern that was done. What is your preference of pattern?

Reply:
I specified `null` in this case, as that is what the attribute is when false.

Reply:
Perfect, thanks.